network/tests/test_network.py

```diff
@@ -125,6 +187,15 @@ def test_cx_state_mocked(is_linux, aggregator, check):
             aggregator.assert_metric(metric, value=value)
 
 
+@mock.patch('datadog_checks.network.network.Platform.is_linux', return_value=True)
```

Comment:
Can you also test with `collect_count_metrics: true` ?

Reply:
The test is already done with `collect_count_metrics` https://github.com/DataDog/integrations-core/pull/11170/files#diff-bdee1557553b0662687ec083f65fd639f102682d8f8da248fa8744acbc34f7f5R193
It's directly passed as parameter so there's no need to mock it.